### PR TITLE
[formrecognizer] test cleanup

### PR DESCRIPTION
**Pull request by kristapratico**
Resolves https://github.com/Azure/azure-sdk-for-python/issues/18642 https://github.com/Azure/azure-sdk-for-python/issues/14300

### CODE REVIEW

File: sdk/formrecognizer/azure-ai-formrecognizer/tests/test_invoice.py [hunk 1]
```diff
@@ -295,8 +295,8 @@ def test_invoice_multipage_pdf(self, client):
 
     @FormRecognizerPreparer()
     @GlobalClientPreparer()
-    def test_invoice_pdf_include_field_elements(self, client):
-        with open(self.invoice_pdf, "rb") as fd:
+    def test_invoice_jpg_include_field_elements(self, client):
```
Comment (iscai-msft) [inline sdk/formrecognizer/azure-ai-formrecognizer/tests/test_invoice.py:298]: out of curiosity, why are you changing pdf tests to jpg?

**Review by iscai-msft — APPROVED** [review]

File: sdk/formrecognizer/azure-ai-formrecognizer/tests/test_invoice.py (hunk 1 above)
Comment (kristapratico) [reply sdk/formrecognizer/azure-ai-formrecognizer/tests/test_invoice.py:298]: Switching to the sample that the service team provided which actually returns all the fields for invoices. It's a jpg; we still have tests that test the pdf form.

File: sdk/formrecognizer/azure-ai-formrecognizer/tests/test_identity_documents.py [hunk 2]
```diff
@@ -198,8 +198,7 @@ def test_identity_document_jpg(self, client):
         # check dict values
         self.assertEqual(id_document.fields.get("LastName").value, "TALBOT")
         self.assertEqual(id_document.fields.get("FirstName").value, "LIAM R.")
-        # FIXME service error when reading the license number returns 'LICWDLACD5DG'
-        # self.assertEqual(id_document.fields.get("DocumentNumber").value, "WDLABCD456DG")
+        self.assertEqual(id_document.fields.get("DocumentNumber").value, "WDLABCD456DG")
```
Comment (catalinaperalta) [inline sdk/formrecognizer/azure-ai-formrecognizer/tests/test_identity_documents.py:201]: glad to see this is fixed!

File: sdk/formrecognizer/azure-ai-formrecognizer/tests/test_invoice_async.py [hunk 3]
```diff
@@ -260,31 +260,6 @@ def callback(raw_response, _, headers):
 
         self.assertFormPagesTransformCorrect(returned_model.pages, read_results, page_results)
 
-    @FormRecognizerPreparer()
-    @GlobalClientPreparer()
-    async def test_invoice_pdf(self, client):
```
Comment (catalinaperalta) [inline sdk/formrecognizer/azure-ai-formrecognizer/tests/test_invoice_async.py:265]: Just wondering why this test was removed?

**Review by catalinaperalta — APPROVED** [review]
Looks great!

File: sdk/formrecognizer/azure-ai-formrecognizer/tests/test_invoice_async.py (hunk 3 above)
Comment (kristapratico) [reply sdk/formrecognizer/azure-ai-formrecognizer/tests/test_invoice_async.py:265]: Sorry should have called that out - it was a duplicate test.

Comment (catalinaperalta) [reply sdk/formrecognizer/azure-ai-formrecognizer/tests/test_invoice_async.py:265]: Ah got it. Thanks for the info! :)

**Pull request merged**